tasks/26_average-mark/average-mark.js

```diff
@@ -0,0 +1,3 @@
+export function getAverage(arr) {
+  return arr.reduce((sum, a) => sum+a)/arr.length >> 0;
```

Comment:
Why do we need `>> 0`?

Reply:
The average should be rounded down to the nearest integer, I guess Math.floor() would be better after all, even for negative numbers